charts/scalardl/templates/ledger/deployment.yaml

```diff
@@ -109,13 +109,28 @@ spec:
           - secretRef:
               name: "{{ .Values.ledger.secretName }}"
           {{- end }}
+          startupProbe:
+            exec:
+              command:
+              - /usr/local/bin/grpc_health_probe
+              - -addr=:50051
+            failureThreshold: 60
+            periodSeconds: 5
```

Comment:
I set the timeout value to 300sec (`failureThreshold * periodSeconds = 300sec`). If the pod won't be `Ready` state after 300sec, Kubernetes terminates the pod.